app/controllers/facility_reservations_controller.rb

```diff
@@ -123,8 +123,8 @@ def new
   def create
     @instrument =
       current_facility.instruments.find_by!(url_name: params[:instrument_id])
-    @reservation =
-      @instrument.admin_reservations.new(params[:reservation].merge(split_times: true))
+    @reservation = @instrument.admin_reservations.new
+    @reservation.assign_times_from_params(params[:reservation])
```

Comment:
This ends up throwing out the note field. In normal `Reservations#create` we do both `assign_attributes` and `assign_times_from_params`.